includes/class-frontend-render-form.php

```diff
@@ -307,7 +307,7 @@ public function render_form( $form_id, $post_id = null, $atts = [], $form = null
             return;
         }
 
-        if ( ! wpuf_user_has_roles( $this->form_settings['roles'] ) ) {
+        if ( 'true' == $this->form_settings['role_base'] && ! wpuf_user_has_roles( $this->form_settings['roles'] ) ) {
```

Comment:
@hasinur1997 use `wpuf_validate_boolean` for boolean testing like 
```
wpuf_validate_boolean( $this->form_settings['role_base'] )
```
No matter what, do not use double comparison. Use triple comparison `===` in both PHP and JavaScript.